Helper/Order.php

```diff
@@ -888,15 +888,7 @@ public function deleteOrderByIncrementId($displayId)
         $order = $this->getExistingOrder($incrementId);
 
         if (!$order) {
-            throw new BoltException(
-                __(
-                    'Order Delete Error. Order does not exist. Order #: %1 Immutable Quote ID: %2',
-                    $incrementId,
-                    $quoteId
-                ),
-                null,
-                CreateOrder::E_BOLT_GENERAL_ERROR
-            );
+            return;
```

Comment:
do we log at all in our plugins? if so can we please add a log line here

Reply:
added bugsnag log